commit 385d4a876b7368171b38d2a0e1960272efe74391 (refs/changes/60/2560/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-09-13T14:01:15-07:00 (2 years, 1 month ago)
    
    RICHMOND-132 Mantis Shrimp Mk. II has incorrect product id
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Reviewed by: Phillip Steinbachs <phillip.steinbachs@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/allocations.md b/allocations.md
index 5c3da82..d1b38af 100644
--- a/allocations.md
+++ b/allocations.md
@@ -24,7 +24,7 @@ which are used as part of SMBIOS and other product naming.
 | Tenderloin A | JCP 2101 | 600-0015 |
 | Tenderloin B | JCP 2201 | 600-0016 |
 | Tenderloin C | JCP 2102 | 600-0017 |
-| Mantis Shrimp MK. II | JSP 7001 | 600-0018 |
+| Mantis Shrimp MK. II | JSP 6001 | 600-0018 |
 | Tenderloin A / Reduced Storage | JCP 2101 | 600-0019 |
 | Tenderloin C / Reduced Storage | JCP 2102 | 600-0020 |
 | Priest River A | JCP 9001 | 600-0021 |
